kore/src/Kore/Repl/Data.hs

```diff
@@ -62,6 +64,14 @@ data ReplCommand
     -- ^ Show the first preceding branch
     | ShowChildren !(Maybe Int)
     -- ^ Show direct children of node
+    | ShowLabels
```

Comment:
If you look at `omit`, the way it works is:
```
omit         shows exclusion list
omit str     if str exists, removes from list, otherwise adds it
```

Would this work for labels:
```
label                   shows label list
label str               goes to line
labeladd str int        adds label
labeldel str            removes label
```

Makes parsing a bit harder, but easier for the user I think. Also, when we get tab completion, it might be easier if they can go "label[tab]" to find label-related commands.

Also, if you see  a nice way to merge it even further, let's discuss. Maybe we could merge them all into a single command like

```
label          -- shows labels
label +todo    -- adds todo label to current node
label +todo 10 -- moves todo label to node 10
label          -- shows labels
label -todo    -- removes todo label
```

This doesn't mean we need to change this type actually. We can still parse it on the same structure, it's only the parser that needs to change.